samza-api/src/test/java/org/apache/samza/operators/descriptors/TestExpandingInputDescriptor.java

```diff
@@ -37,15 +37,13 @@ public void testAPIUsage() {
     ExampleExpandingOutputDescriptor<Integer> output1 = expandingSystem.getOutputDescriptor("output1", new IntegerSerde());
 
     input1
-        .withBootstrap(false)
+        .shouldBootstrap()
```

Comment:
Is it intentional to change from setting to false to setting to true for bootstrap and reset offset? Since this is just a "compile" test, it seems ok, but wanted to double check.

Reply:
Yeah, this was just for testing the API, so should be OK.